### DISCUSSION TITLE
Proposal: <podcast:disclosure> for affiliates, AI, sponsorships, etc.

### DISCUSSION THREAD

**Discussion by theDanielJLewis** [Podcastindex-org/podcast-namespace#669] [Ideas]
To be in compliance with current and possibly upcoming laws, podcasters must include certain disclosures based on their content. For example:

- Anyone paid to use or talk about a product must make it obvious that they are compensated. 
- Affiliates must disclose that they earn commissions. 
- Certain AI usage will probably be required by law to be disclosed.
- If the guest paid to be in the podcast, that must be disclosed.

While such disclosures must be prominent and in close proximity to the information, which means always being within the content of the podcast episodes, I think they should also be disclosed within the feed.

And this would help address #663 and other concerns over AI use for generating content, maybe voices, and maybe realistic-looking images.

## Examples

Here's a rough proposal for the `<podcast:disclosure>` tag that could be both item- and channel-level, depending on the context, allowing multiple instances for each necessary disclosure.

```xml
<podcast:disclosure type="compensation" context="links">Some of the products/services in this episode use affiliate links that compensate the podcaster for qualifying purchases.</podcast:disclosure>
<podcast:disclosure type="compensation" context="content">This episode is sponsored by its guest.</podcast:disclosure>
<podcast:disclosure type="compensation" context="ad">This episode contains paid ad placements.</podcast:disclosure>
<podcast:disclosure type="ai" context="content">This episode contains AI-generated content.</podcast:disclosure>
<podcast:disclosure type="ai" context="images">This episode uses realistic AI-generated images.</podcast:disclosure>
<podcast:disclosure type="association" context="content">I am an advisor to XYZ Acme.</podcast:disclosure>
<podcast:disclosure type="license" context="audio">"Best Song Ever" © 2024 Daniel J. Lewis. Used by permission.</podcast:disclosure>
```

## Options

- `type` attribute: Defines the category of disclosure from a specific list of possibilities, such as `compensation` for any kind of financial or similar transaction associated with the specific content, `ai` for disclosure-necessary generative AI use, `association` for when someone might be speaking about a company with which they are associated (on the board, employed, own shares, etc.), and `license` for stuff that is used by special license or permission.
- `context` attribute (or a better name): Defines _what_ content the disclosure is covering from a specific list of possibilities. (I thought of the attribute `content`, but then `content="content"` seems strange.) These options would include, at a minimum, `links` for any links mentioned in the episode or its notes, `content` for whole sections of content (like a paid guest appearance, a sponsored segment, a sponsored review, etc.), `ad` for disclosing that the episode _does_ contain paid ads, `images` for use of images (cover art, chapter art, in notes, etc.), and `audio` (or maybe more specifically `music`, `sound-effects`, and `soundbites`) for any additional audio (I recognize this needs to be distinguished from something like AI-generated voices).
- Content: Open short text, ideally with the publishing tool's suggesting common (and lawyer-approved) text, or allowing the podcaster to adjust it for their needs.

## Why?

This single tag can address multiple needs for legal and ethical compliance and be easily update for future needs.

What do you think?

**Comment by jamescridland** [comment]
Tags with legal implications, like this one, really need to be driven by Apple+Spotify. Without those app's support, these disclosures won't be visible for the majority of podcast listeners, and so therefore won't be a solution.

Given that YouTube does have a requirement for flagging "contains paid content", something that its current RSS ingestion feature fails to import, this might be a good start to gain take-up. Similarly, Apple has a requirement for AI to be prominently disclosed in certain circumstances. But these folk need to drive this as an idea.

**Reply by samsethi** [reply]
In TrueFans we have various paid content items - audiobooks, music, courses, films etc. 

<img width="1237" alt="Cursor_and_TrueFans___Shows___The_Three_Little_Sheepdogs___Their_Journey_Home" src="https://github.com/user-attachments/assets/2cf0ae0b-f29b-4224-98f4-4eedc61905ec" />

Spotify refers to "contains paid content" meaning subscription or paywall.  The content we have in TrueFans can be labelled paid content but how should we do it? We can add this to the Creators Dashboard but what is the podcast tag? /cc @theDanielJLewis 

<img width="721" alt="spotify_contains_paid_content_label_-_Google_Search" src="https://github.com/user-attachments/assets/10780d1c-3181-400f-8826-80e7ae3fef63" />

**Comment by Dwev** [comment]
I think this is an excellent idea for both AI and compensation disclosures.  It still requires the creator to be honest and tag correctly, but with the way pure AI-generated content is saturating a lot of platforms, I'm sure this will be needed soon.

**Comment by samsethi** [comment]
TrueFans has already added two options regarding AI.  In the Creators Dashboard you can self-declare a podcast as AI hosted.  We do not care if AI was used to generate the show notes or transcript just the host(s). 

We then show a robot icon in the app UI to indicate AI hosted.  This is similar to the Explict Tag and E icon.   

<img width="1014" alt="TrueFans___Dashboard" src="https://github.com/user-attachments/assets/81301995-6fc2-4e02-a6a6-ee2a21054f37" />

We then allow users in the settings to decide if they want to hear Explicit and/or AI content.  

<img width="344" alt="TrueFans___Settings" src="https://github.com/user-attachments/assets/7b00f694-6cb4-419d-9c22-06eb38fd2800" />

We need to add a **Podcast:AI** tag to RSS like we do the explicit tag.  I don't see this being a hard decision???

<img width="155" alt="Best_Online_XML_Viewer__XML_Formatter__XML_Editor__Analyser__Beautify-Beautifier__Minify__Tree_structure" src="https://github.com/user-attachments/assets/308a6376-74c0-4ec1-ac54-e97cb03962c9" />

**Comment by theDanielJLewis** [comment]
I want to bring attention back to this because we really don't need separate disclosure tags for separate disclosures. Please, let's make a single tag that can handle all disclosures. That would be much cleaner, optimized, it won't waste space, it will be easier to support, and it is future friendly.

**Reply by redimongo** [reply]
Am going to implement this as is into PodToo. 

I can only recommend it to our creators, but can at least advocate for it. 

As long as it doesn't penalise our creators. 

The only thing I would like to see is a standard list of content for AI. 

I'll put a list together later and edit this comment.

**Comment by nathangathright** [comment]
Podcasters who move disclosures from their show notes to this tag will be effectively hiding that disclosure from the vast majority of listeners. I’m struggling to understand how something like this would build momentum without one of the major apps incentivizing participation.

**Reply by theDanielJLewis** [reply]
I never said they would have to _move_ their disclosures. But this provides a specific place to hold the data, and it's more inclusive and future-proof than adding an AI-only disclosure somewhere else.

**Reply by nathangathright** [reply]
I’m not trying to make a case in favor of my proposal here. I’m saying this proposal either needs to find a major advocate, or it needs to persuade podcasters, hosts, and apps that it can provide value even at low adoption rates.

**Comment by samsethi** [comment]
Getting ahead of regulation might be a smart thing to do.  This 'disclosure tag' might be a way to group everything in one tag but I think it is overkill for now.  I also don't think creators will care enough until there is a legal reason to do so. However I do think the ONLY thing users would like to know is if the voice they hear is a human or an AI voiced host.  #turingtest

I don't think users cares if the podcast script was created using ChatGPT, or the episode title, chapters or transcript were generated using AI or if the coverart was made using AI.  None of that matters to users. 

As I suggested before a simple AI toggle to indicate the host is or isn't an AI is all we need for now.  Make it voluntary to begin with like the explicit tag.  And when tech companies catch up and there is an automated way for a host to detect the AI voice.

**Reply by theDanielJLewis** [reply]
> I also don't think creators will care enough until there is a legal reason to do so. 

Well, there _is_ a legal reason to disclose already: compensation. In the USA, we have the FTC saying that people must disclose if they're compensated for endorsing or featuring products, people, or services.

> I don't think users cares if the … episode title, chapters or transcript were generated using AI or if the coverart was made using AI. None of that matters to users.

I agree with that, and I don't think any AI disclosure is required here because this is, mostly, based on the podcaster's own content. But I _do_ think that creating the content from an LLM would matter to the audience, because LLM-created content is often lower quality and not original.

As for when to disclose AI usage, I think it should be when the results could be confused for reality. For example, showing imagery that might be thought real (like using a known figure or about a known event or location). Or when a voice sounds real but there's actually no human there.

**Reply by redimongo** [reply]
> I don't think users cares if the podcast script was created using ChatGPT, or the episode title, chapters or transcript were generated using AI or if the coverart was made using AI. None of that matters to users.

Its not about whether users care about it or not, do users really care about how their personal data is being used by an app? or how an app uses your location? In many cases people don't scroll down on the app store listing and make buying choices based on the disclosers in the store.

This discloser could bring is inline with TV and Movies like a credit screen that tells you what tools I used, what sponsors I had, which AI tool was used to get the end result.

The bigger issue is people will not disclose AI voices because companies like Youtube is already cracking down on AI made slop/content and saying "we wont be paying out" if everyone "HAD TO" disclose this information it could be a very fast IAB ban on ads in AI generated podcasts.

So where we sit, is we kinda need to teach people that saying you have used AI to make transcripts or your chapters or maybe a jingle or two is a small first step. I get its not the end goal that many would like and may not solve the problem that some players want which is "I want to know if this is an AI voice or not." but kinda got to approach this as a stepping stone and get users use to disclosing it.

**Comment by theDanielJLewis** [comment]
@jamescridland, in the latest PWR, you said this proposes a tag that can't be programmatically understood. But I beg to differ. The programmatically available information is in the tag's attributes.

For example, the following indicates that AI was used to generate content.

```xml
<podcast:disclosure type="ai" context="content">…</podcast:disclosure>
```

Yes, I think the assumption for `type="ai"` should be that something was _generated_, because there's no need to indicate something was enhanced with AI (like social posts, titles, chapters, audio processing, etc.).

Aside from the AI discussion, the much bigger _already-present_ need I want everyone to remember is that podcasters are already required to disclose when they are compensated for including something. That's why this is a generic "disclosure" tag instead of anything AI-specific and limited to only a single use.

**Comment by redimongo** [comment]
As I commented in September we said we would integrate this tag, that integration when we spoke to creators didn't really go down well. The main feedback from creators we spoke to said the following: "I don't have time to fill out more boxes". While this may not be all creators and I am sure most creators would disclose, I was thinking how can we make AI disclose this.

What I have been working on is away that we can highlight a score 0-100% based on AI detection software that we have developed and tested, which is included in our AI transcription tool. 

basically when it makes the transcription its looking for anything AI (music, voices, even if the script sounds AI) then it calculates how much of the content was AI, at the moment I haven't labelled each like (x voice is ai), (x music is ai) - it would slow the script down.

But what we can offer is 

```
<podcast:disclosure type="ai" context="content">PERCENTAGE SCORE</podcast:disclosure>
```

@theDanielJLewis I don't know how you feel about that idea?

**Reply by jamescridland** [reply]
I can't see why any podcast host would implement this, I'm afraid. Unless they're told to do so by platforms.

Using AI to look for AI is... interesting.

**Reply by redimongo** [reply]
> I can't see why any podcast host would implement this, I'm afraid. Unless they're told to do so by platforms.
> 
> 
> 
> Using AI to look for AI is... interesting.

I agree with @jamescridland that I also don't see others implementing this. But someone needs to at least be looking like we are trying to solve a problem that the community is saying we have. 

On our tests - it has successfully picked up every part of a podcast that uses AI voices or AI music. So a score system seems the most effective solution based on transcription length + speaks etc. 

I don't really have another solution that would make it more clear to a player or the index that X is ai generated, and I see @daveajones working hard to try and remove slop but that's human intervention and time consuming (sure i think he will make the AI do it but that's costly to the index to run 24x7 and with the speed new feeds pop up)

As PodToo roles out our AI servers (with our own GPUs) it for us really won't cost too much to handle this on level of customers we have.

Use case testing results have shown our system can do 80minutes of audio recording within around 11 minute that is returning transcript, speaker labels (with names), chapters, ai score, social media posts, time codes to generate suggested audiograms / videograms. 

But that clearly is us putting it in our workflow for our clients including our Cloud customers. 

I am happy to be that host provider that says every episode now includes an AI score.

Will that be helpful on an index level? I have no idea. 

I would even be willing to work with Dave to integrate our ai score system into new 3rd party feeds that get fed into the index by other providers. But clearly we can't insert it into the feed, but could be a label added to the index for those who want to read it (just an idea).

**Comment by kookster** [comment]
Noting that Apple has added AI Transparency tags to Apple Music - so this is perhaps a form of endorsement by that platform, if not yet for the podcasts content type:
https://help.apple.com/itc/musicspec/#/itccdd938379:~:text=Track%20Artificial%20Intelligence

**Reply by Marzal** [reply]
Thanks for the link.

@nathangathright use it for this new proposal:
https://github.com/Podcastindex-org/podcast-namespace/discussions/753

PD: I miss the issue functionality of auto-referencing / cross-linking that discussions don't have

**Comment by theDanielJLewis** [comment]
I'd still like to offer this as the solution to handle both the _recommendation_ to disclose use of AI, as well as the _legally requirement_ to disclose compensation/paid content/sponsorship.

A single disclosure tag would be more appropriate. Plus, it would also address #415.
